quarkus/runtime/src/main/resources/application.properties

```diff
@@ -30,7 +30,7 @@ quarkus.package.include-dependency-list=false
 quarkus.devservices.enabled=false
 
 # We want to expose non-application paths (e.g. health) at the root path
-quarkus.http.non-application-root-path=${quarkus.http.root-path}
+quarkus.http.non-application-root-path=/${quarkus.http.root-path}
```

Comment:
I'm not 100% sure about it. It might (and will) lead to scenarios when we have a leading double-slash, e.g. `//auth`. While this might work now, I feel it is not a correct value and might break in the future. Or am I wrong?

I think a correct fix might be we just add a leading `/` if not present?

Reply:
Ok, sounds good.